CMakeLists.txt

```diff
@@ -1050,7 +1050,7 @@ endif ()
 
```

Comment:
I don't see the dynamorio submodule being updated?  So this is passing extra args to functions that don't take them?  I guess CMake doesn't complain and things pass but that's not what we want.  We want to update the submodule.

Reply:
I think I just did, please double-check if that was right (just git added dynamorio to latest)